height: 50.0
width: 50.0
cell_size: 0.5
DEBUG: true
floor_filter: true
heightmap_threshold: 0.15
max_height: 2.5
dis_VW: 10.0